common_test.go

```diff
@@ -0,0 +1,23 @@
+package gsclient
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_isValidUUID(t *testing.T) {
+	validUUIDTest := make([]uuidTestCase, len(uuidCommonTestCases))
+	copy(validUUIDTest, uuidCommonTestCases)
+	validUUIDTest = append(validUUIDTest, uuidTestCase{
```

Comment:
This name `validUUIDTest` is misleading. I expect a valid UUID to be tested, however, 'abc-123` is invalid. I would add two tests `validUUIDTest` and `invalidUUIDTest`.